selfdrive/modeld/modeld.cc

```diff
@@ -191,6 +188,18 @@ int main(int argc, char **argv) {
   model_init(&model, device_id, context);
   LOGW("models loaded, modeld starting");
 
+  bool main_wide_camera = false;
+  while (!do_exit) {
+    auto streams = VisionIpcClient::getAvailableStreams("camerad", false);
+    if (!streams.empty()) {
+      main_wide_camera = (streams.size() == 1 && streams.count(VISION_STREAM_WIDE_ROAD) == 1);
```

Comment:
this should also handle the case of a driver cam + road cam, so the check for size == 1 doesn't work